X509-XWING-2024.asn

```diff
@@ -0,0 +1,58 @@
+XWing-KEM-2024
+   { iso(1) member-body(2) us(840) rsadsi(113549) pkcs(1)
+     pkcs-9(9) smime(16) modules(0) id-mod-XWing-kem-2024(TBD) }
```

Comment:
This is a different registry from where it's allocated for [ML-KEM](https://github.com/lamps-wg/kyber-certificates/blob/8871c86af729f39c0c098cfad408a6c849264f7f/X509-ML-KEM-2024.asn#L2). Why the difference?

Reply:
We usually allocate the module OIDs for CMS in 1.2.840.113549.1.9.16.0, and we allocate the modules OIDs for PKIX-related things in 1.3.6.1.5.5.7.0.  Either way, you get a unique value.